### ISSUE TITLE
findMeanSD.py doesn't create output but no error message

### ISSUE THREAD

**Issue by thjkral** [jigold/zCall#7]
When findMeanSD.py is executed, it creates a file which only contains a header but no additional lines. To make things more confusing, there are no errors given so I don't know what went wrong.

What is a possible cause for this?

**Comment by delphineDB** [comment]
Hi,
I meet the same problem. Did you find the solution?

Thanks

**Comment by TaKaRa001** [comment]
Hello,
I met the same problem, as well. @thjkral @delphineDB: How did you solve this issue in the end?
Thank you!

**Comment by brettvanderwerff** [comment]
@TaKaRa001 I am at the same point with the same issue. Did you come to a solution. I may contact the author soon but this is a pretty old repo and looks like it might be abandoned

**Comment by TaKaRa001** [comment]
@brettvanderwerff Yes, I found a solution. I didn´t exclude samples I zeroed in GenomeStudio. Additionally, make sure you just include samples with a CallRate of at least 98%. When I checked relatedness at a later stage I had problems with these samples.
